frontend/public/style/_forms.scss

```diff
@@ -6,3 +6,7 @@
   content: '*';
   padding-left: 3px;
 }
+
+.create-form__subsection {
```

Comment:
This doesn't really follow BEM. `create-form__subsection` would be an element under the `create-form` block, but we don't have a block with the class.

http://getbem.com/naming/

I'd probably just call this `co-form-subsection` to be used in any form (not just create forms).